Instructions on how to deploy a branch to CG-VM1 in a collapsible section of PR template (#3101)
Clinical-Genomics/scout: .github/PULL_REQUEST_TEMPLATE.md

OR
This PR marks a new Scout release. We apply semantic versioning. This is a major/minor/patch release for reasons.

> **Prepare for testing**

Reviewer: Maybe add a Stockholm node header?

Author: Good suggestion, I'll add it!

OR
This PR marks a new Scout release. We apply semantic versioning. This is a major/minor/patch release for reasons.

<details>
<summary>Testing on cg-vm1 server (Clinical Genomics Stockholm)</summary>

**Prepare for testing**
1. Make sure the PR is pushed and available on [Docker Hub](https://hub.docker.com/repository/docker/clinicalgenomics/scout-server-stage)
1. `ssh your.email@cg-vm1.scilifelab.se`
1. `sudo -iu hiseq.clinical`
1. `ssh localhost`

> 1. Find which scout branch (old_branch) is currently deployed on cg-vm1: `podman ps`

Reviewer 2: ```suggestion
1. (optional) Find out which scout branch is currently deployed on cg-vm1: `podman ps`
```

Author: Good solution! 👍🏻

> 1. Stop the service with current deployed branch: `systemctl --user stop scout@<old_branch>`

Reviewer 2: ```suggestion
1. Stop the service with current deployed branch: `systemctl --user stop scout.target`
```

1. Start the scout service with the branch to test: `systemctl --user start scout@<this_branch>`

> 1. Make sure the branch is deployed: `systemctl --user status scout@<this_branch>`

Reviewer 2: ```suggestion
1. Make sure the branch is deployed: `systemctl --user status scout.target`
```
